config/secrets.yml

```diff
@@ -0,0 +1,25 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Is this file being used?

Reply:
Good nudge thanks — in routes.rb and (with ad32c195) in secret_token.rb:

```
config/routes.rb
6:Sidekiq::Web.set :session_secret, Rails.application.secrets[:secret_key_base]

config/initializers/secret_token.rb
1:Bikeindex::Application.config.secret_key_base = Rails.application.secrets[:secret_key_base]
```